meta-lmp-base/recipes-security/optee/optee-fiovb_git.bb

```diff
@@ -1,6 +1,6 @@
 SUMMARY = "OP-TEE Foundries.IO Verified Boot Client Application"
 HOMEPAGE = "https://github.com/foundriesio/optee-fiovb"
-LICENSE = "GPLv2"
+LICENSE = "GPL-2.0-only"
```

Comment:
Another one that seems wrong, I believe it should be BSD-2-Clause. (https://github.com/foundriesio/optee-fiovb/blob/master/LICENSE).